applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_volume.c

```diff
@@ -6,7 +6,6 @@
 
 #include "bt_volume.h"
 
-#include <stdint.h>
```

Comment:
I assume you have compiled for BIS and CIS before pushing up? Since there is no CI running on this

Reply:
Yes I have tried compiling for both CIS and BIS